.markdownlint.json

```diff
@@ -1,6 +1,6 @@
 {
     "default": true,
     "MD013": {
-        "line_length": 120
+        "line_length": 140
```

Comment:
This is needed as adding inline comments to a yaml its not very flexible 😄

Reply:
Haha nice